sound/soc/codecs/ma120x0p.c

```diff
@@ -1064,14 +352,13 @@ static int ma120x0p_probe(struct snd_soc_component *component)
 
 	// Enable audio limiter
 	ret = snd_soc_component_update_bits(component,
-		ma_audio_proc_limiterenable__a,
-		ma_audio_proc_limiterenable__mask, 0x40);
+		ma_audio_proc_limiterenable__a, ma_audio_proc_limiterenable__mask, 0x40);
 	if (ret < 0)
 		return ret;
 
 	// Set lim attack to fast
 	ret = snd_soc_component_update_bits(component,
-		ma_audio_proc_attack__a, ma_audio_proc_attack__mask, 0x80);
+		ma_audio_proc_attack__a,ma_audio_proc_attack__mask, 0x80);
```

Comment:
Random deletion of whitespace that is required by the coding standard (and general readability).